specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/vmssNetworkInterface.json

```diff
@@ -0,0 +1,389 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2017-03-30"
```

Comment:
2018-12-01

Reply:
VMSS has it's own API version, this is expected

Reply:
Should it be under it's own version folder then?

Reply:
I don't know all the details, there's an open issue for that: https://github.com/Azure/azure-rest-api-specs/issues/1685